services/catalog/rs_server_catalog/user_handler.py

```diff
@@ -180,29 +180,34 @@ def add_user_prefix(  # pylint: disable=too-many-return-statements
     Returns:
         str: The RS server frontend endpoint.
     """
+    new_path = path
+
     if path == "/collections":
-        return CATALOG_COLLECTION
+        new_path = CATALOG_COLLECTION
```

Comment:
Why do you use a `new_path` variable, why can't you just return as before ?

Reply:
It's for the code readability + easier to manipulate in case of debugging.